app/forms/assessor_interface/verify_requestable_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AssessorInterface::VerifyRequestableForm
```

Comment:
```suggestion
class AssessorInterface::RequestRequestableForm
```

Since it's not verifying the requestable, it's marking it as requested.